### PR TITLE
restart machine when no breath

### PR DESCRIPTION
**Pull request by rfire01**

### CODE REVIEW

File: algo.py [hunk 1]
```diff
@@ -157,7 +157,7 @@ def __init__(self, measurements, events):
         self.pressure_slope = RunningSlope(num_samples=7)
         self._config = Configurations.instance()
         self.last_breath_timestamp = None
-        self.current_state = VentilationState.PEEP
+        self.current_state = VentilationState.Exhale
```
Comment (avrahamshukron) [inline algo.py:160]: Explain

Comment (rfire01) [reply algo.py:160]: From what i understand, there are only two states inhale and exhale, im wrong?

Comment (avrahamshukron) [reply algo.py:160]: > @avrahamshukron  From what i understand, there are only two states inhale and exhale, im wrong?

Not wrong. The flow based machine only distinguish between inhale and exhale. Still I think it is better to initialize the state to some neutral value like Calibration.
This way, the first transition (inhale or exhale) will be caught property

**Comment by osherdp** [comment]
@rfire01 @avrahamshukron why pPeak is being reset?
I believe it's not relevant to a no breathing scenario

Also, how do you purpose we test it heavily to make sure it makes our logic more correct?

**Comment by rfire01** [comment]
> @rfire01 @avrahamshukron why pPeak is being reset?
> I believe it's not relevant to a no breathing scenario
> 
> Also, how do you purpose we test it heavily to make sure it makes our logic more correct?

When there is no breathing you want to return a clean state, i.e. if there is a new breath recognized that it will start from the beginning (same as starting the app from the start), so it make sense to also restart the peak (as you don't want the next time it calculated to contain value from the time before no breath happened)

**Pull request merged**